src/tpm/object_tools.c

```diff
@@ -400,12 +400,11 @@ int create_kmyth_object(TSS2_SYS_CONTEXT * sapi_ctx,
               temp_handle);
 
     // Make the newly recreated SRK primary object persistent
-    TSS2_RC rc = Tss2_Sys_EvictControl(sapi_ctx,
-                                       TPM2_RH_OWNER,
-                                       temp_handle,
-                                       &createObjectCmdAuths,
-                                       object_dest_handle,
-                                       &createObjectRspAuths);
+    rc = Tss2_Sys_EvictControl(sapi_ctx,
```

Comment:
TSS2_RC rc was already previously declared in the same scope

Reply:
Does not matter. The rc (return code) is reused.